### PR TITLE
Fixes and improvements for pt-BR translations

### PR DESCRIPTION
**Pull request by gerep**

### CODE REVIEW

**Review by ykzts — DISMISSED** [review]

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
 pt-BR:
   about:
-    about_mastodon: Mastodon é um servidor de rede social <em>grátis, e open-source</em>. Uma alternativa <em>descentralizada</em> ás plataformas comerciais, que evita o risco de uma única empresa monopolizar a sua comunicação. Escolha um servidor que você confie &mdash; qualquer um que escolher, você poderá interagir com todo o resto. Qualquer um pode ter uma instância Mastodon e assim participar na <em>rede social federada</em> sem problemas.
+    about_mastodon: Mastodon é um servidor de rede social <em>grátis e open-source</em>. Uma alternativa <em>descentralizada</em> às plataformas comerciais, que evita o risco de uma única empresa monopolizar a sua comunicação. Escolha um servidor que você confie &mdash; qualquer um que escolher, você poderá interagir com todo o resto. Qualquer um pode ter uma instância Mastodon e assim participar na <em>rede social federada</em> sem problemas.
```
Comment (Gargron) [inline config/locales/pt-BR.yml]: Could you check the new landing page and adjust this translation accordingly?

Comment (gerep) [inline config/locales/pt-BR.yml]: Sure! I just noticed that they are really different. I will make the updates ;)

**Comment by sorin-davidoi** [comment]
I guess #4185 can be included here?

**Review by nightpool — CHANGES_REQUESTED** [review]
needs review from another Brazilian translator/user.

**Comment by ykzts** [comment]
The original repository has been lost.

@gerep I close this Pull Request. Please reopen this pull request again when you resume.

**Pull request closed without merging**